specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-02-14-privatepreview/postgresql.json

```diff
@@ -1164,10 +1250,13 @@
             "modelAsString": true
           }
         },
-        "standbyCount": {
-          "type": "integer",
-          "format": "int32",
-          "description": "stand by count value can be either 0 or 1"
+        "maintenanceWindow": {
+          "$ref": "#/definitions/MaintenanceWindow",
+          "description": "Maintenance window of a server."
+        },
+        "haEnabled": {
+          "type": "string",
```

Comment:
Limit this to 'enabled' and 'disabled':
      "enum": [
        "Enabled",
        "Disabled"
      ],
      "x-ms-enum": {
        "name": "SslEnforcementEnum",
        "modelAsString": false
      }

Reply:
Defined as enum to restrict the values.